packages/tradershub/src/constants/regulators-modal-content.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable sort-keys */
```

Comment:
@sergei-deriv I believe this content is just useful for regulation table, so better to put it beside the component rather than here